README.md

```diff
@@ -41,26 +43,31 @@ Tracked Image
 
-Together <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.datasets.html">deepcell.datasets</a></tt> and <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.applications.html">deepcell.applications</a></tt> provide an accessible entrypoint to deep learning for biologists. The datasets module contains a variety of annotated datasets that can be used as training data. Additionally, the applications package initializes a set of models complete with the option to initialize with pre-trained weights.
+`deepcell-tf` contains two modules that greatly simplify the development and usage of deep learning models for single cell analysis. The first is <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.datasets.html">deepcell.datasets</a></tt>, which a collection of biological images that have single-cell annotations. These data include live-cell imaging movies of fluorescent nuclei (approximately 10,000 single-cell trajectories over 30 frames), as well as static images of whole cells (both phase and fluorescence images - approximately 75,000 single cell annotations). The second is <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.applications.html">deepcell.applications</a></tt>, which contains pre-trained models (fluorescent nuclear and phase/fluorescent whole cell) for single-cell analysis. Provided data is scaled so that the physical size of each pixel matches that in the training dataset, these models can be used out of the box on live-cell imaging data. We are currently working to expand these modules to include data and models for tissue images. Please note that they may be spun off into their own GitHub repositories in the near future.
 
-## DeepCell for Developers
+## DeepCell-tf for Developers
 
-DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing. If using GCP, there are [pre-built images](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) which come with CUDA, docker, and nvidia-docker pre-installed. Otherwise, you will need to install [docker](https://docs.docker.com/install/linux/docker-ce/debian/), [nvidia-docker](https://github.com/NVIDIA/nvidia-docker), and [CUDA](https://developer.nvidia.com/cuda-downloads) separately.
+`deepcell-tf` uses `nvidia-docker` and `tensorflow` to enable GPU processing. If using GCP, there are [pre-built images](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) which come with CUDA, docker, and nvidia-docker pre-installed. Otherwise, you will need to install [docker](https://docs.docker.com/install/linux/docker-ce/debian/), [nvidia-docker](https://github.com/NVIDIA/nvidia-docker), and [CUDA](https://developer.nvidia.com/cuda-downloads) separately. These instructions set up a docker container so that you can directly change the `deepcell-tf` library itself and have those changes reflected within the container
```

Comment:
Would it make more sense to include this information about pre-built instances in the getting started section?

If a relatively novice person tries to get this up and running, they're going to get sent to the nvidia-docker link in the getting started section and instantly get stuck in CUDA library dependency mess.

Reply:
I see @ngreenwald's point here. The historical context is that we have always introduced new users to deepcell-tf under the assumption that they had access to a server where all the standard dependencies are already in place (Docker, Jupyter, and all the correct drivers). While we've had users who prefer to get started with VMs, that has not been our preferred introduction. Additionally, our docs are structured in such a way as to provide minimal starting information on the READMEs with a pointer to RTD. This was a design decision to keep the READMEs as simple as possible. That being said, both are still under construction and in need of improvements.

I'd welcome input from @willgraf and @msschwartz21 before proceeding

Reply:
I agree with @ngreenwald that our existing Getting Started section makes several non-trivial assumptions about the user's local environment and their access to GPU. I also think that we should still continue to prioritize keeping READMEs as simple as possible.

Perhaps as a compromise, we can make the following changes starting at line 46:
> The fastest way to get started with `deepcell-tf` is to run one of our existing images from Docker Hub. To do so locally, you need access to a GPU with [docker](https://docs.docker.com/install/linux/docker-ce/debian/), [nvidia-docker](https://github.com/NVIDIA/nvidia-docker), and [CUDA](https://developer.nvidia.com/cuda-downloads). Alternatively, Google Cloud Platform (GCP) offers [prebuilt virtual machines](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) preinstalled with Cuda, docker and nvidia-docker.

Then in the developer docs we can just focus on building a docker image for development.

Reply:
We currently maintain 2 images for each release, the CPU and GPU builds.

Maybe we can introduce the "easiest way" as using the CPU build, which should run anywhere, and we can have another section about GPU usage where we go into more detail about `nvidia-docker` and on-prem vs cloud GPU servers.

Reply:
Added mention of CPU and VM alternatives in "Getting Started"